File: app/src/main/java/com/linxiao/quickdevframework/sample/widget/WidgetsGuideFragment.java
package com.linxiao.quickdevframework.sample.widget;

import android.content.Intent;
import android.graphics.Color;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;

import com.linxiao.framework.architecture.BaseFragment;
import com.linxiao.framework.widget.TextDrawable;
import com.linxiao.quickdevframework.R;
import com.linxiao.quickdevframework.sample.mvvm.CaptchaActivity;

import butterknife.BindView;
import butterknife.ButterKnife;
import butterknife.OnClick;

/**
 * Created by linxiao on 2017/2/12.
 */
public class WidgetsGuideFragment extends BaseFragment {


    @BindView(R.id.ivTextDrawable)
    ImageView ivTextDrawable;

    @Override
    protected void onCreateContentView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        setContentView(R.layout.fragment_widgets_guide, container);
        ButterKnife.bind(this, getContentView());
        TextDrawable drawable = TextDrawable.createRound();
        drawable.setBackgroundColor(Color.BLACK);
        drawable.setTextColor(Color.WHITE);
        drawable.setText("辣鸡");
        ivTextDrawable.setImageDrawable(drawable);
    }

    @OnClick(R.id.btnMVVMSample)
    public void onMVVMSampleClick(View v) {
        startActivity(new Intent(getActivity(), CaptchaActivity.class));
    }
}

File: app/src/main/java/com/linxiao/quickdevframework/sample/widget/WidgetsGuideFragment.kt
package com.linxiao.quickdevframework.sample.widget

import android.content.Intent
import android.graphics.Color
import android.os.Bundle
import android.view.View
import com.linxiao.framework.architecture.SimpleViewBindingFragment
import com.linxiao.framework.common.DensityHelper
import com.linxiao.framework.widget.TextDrawable
import com.linxiao.quickdevframework.databinding.FragmentWidgetsGuideBinding
import com.linxiao.quickdevframework.sample.mvvm.CaptchaActivity

/**
 *
 * @author lx8421bcd
 * @since 2017-02-12
 */
class WidgetsGuideFragment : SimpleViewBindingFragment<FragmentWidgetsGuideBinding>() {
    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        val drawable = TextDrawable.createRound()
        drawable.backgroundColor = Color.BLACK
        drawable.textColor = Color.WHITE
        drawable.text = "辣鸡"
        drawable.textSize = DensityHelper.dp2px(12f).toFloat()
        drawable.setPadding(4,4,4,4)
        viewBinding.ivTextDrawable.setImageDrawable(drawable)
        viewBinding.btnMVVMSample.setOnClickListener { v: View? -> onMVVMSampleClick(v) }
    }

    fun onMVVMSampleClick(v: View?) {
        startActivity(Intent(activity, CaptchaActivity::class.java))
    }
}
